ios/Podfile.lock

```diff
@@ -12,7 +12,7 @@ PODS:
     - React-jsi (= 0.64.1)
```

Comment:
The changes here are coming from develop (dependencies updates from dependabot where merged without running `pod install`)